src/wallet/wallet2.cpp

```diff
@@ -3015,6 +3020,7 @@ bool wallet2::store_keys(const std::string& keys_file_name, const epee::wipeable
   rapidjson::Document json;
   json.SetObject();
   rapidjson::Value value(rapidjson::kStringType);
+  
```

Comment:
oops

Reply:
Yes, this empty line should not have been added. Removed again.